src/main/java/org/snomed/snowstorm/mrcm/MRCMUpdateService.java

```diff
@@ -222,7 +247,13 @@ private List<ReferenceSetMember> updateAttributeRules(Commit commit, Map<String,
 	}
 
 	private Set<String> getMRCMRefsetComponentsChanged(Commit commit) {
-		BranchCriteria branchCriteria = versionControlHelper.getBranchCriteriaChangesAndDeletionsWithinOpenCommitOnly(commit);
+		BranchCriteria branchCriteria;
+		if (BranchMetadataHelper.isSkipMrcmUpdateServiceCommit(commit)) {
+			branchCriteria = versionControlHelper.getBranchCriteria(commit.getBranch());
+		} else {
+			branchCriteria = versionControlHelper.getBranchCriteriaChangesAndDeletionsWithinOpenCommitOnly(commit);
+		}
+
```

Comment:
If this method is reached via the commit listener, we want to find all members reachable visible to the commit. 

If this method is reached via the manual trigger, we want to find all reachable members visible to the branch.